backend/src/auth/auth.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class AuthService {
+  getHello(): string {
```

Comment:
Maybe we should get rid of this now ahah :laughing:

Reply:
One day it will work!